src/node_options.cc

```diff
@@ -301,6 +333,10 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             "show stack traces on process warnings",
             &EnvironmentOptions::trace_warnings,
             kAllowedInEnvironment);
+  AddOption("--entry-type",
+            "top-level module type name",
```

Comment:
nit: I find this description slightly confusing ... maybe:

`default module syntax supported by application entry-point` ... a little bit verbose, but maybe something like this?

Reply:
attempted improvement in 6dadd5b1b7